pwndbg/gdblib/arch.py

```diff
@@ -7,7 +7,7 @@
 
 # TODO: x86-64 needs to come before i386 in the current implementation, make
 # this order-independent
-ARCHS = ("x86-64", "i386", "aarch64", "mips", "powerpc", "sparc", "arm")
+ARCHS = ("x86-64", "i386", "aarch64", "mips", "powerpc", "sparc", "arm", "armcm")
```

Comment:
I guess the reason we're adding `armcm` here is in case we want to use it with `@OnlyWithArch`? The reason it wasn't initially in this list is that there is no real `armcm` GDB architecture, we just check for ARM architectures ending in "-m":
```
if match == "arm" and "-m" in arch:
    match = "armcm"
```

I'm slightly concerned about having an arch that doesn't directly map to a GDB arch here, but I also don't know a better solution that doesn't require more duplication than it's worth. No need to change anything here, just calling this out.

Reply:
I think I added it here because it was in the pwnlib arch mapping dict.

I'm fine with removing that change.

Btw the ARCHS is only used in this context, jfyi:
<img width="1002" alt="image" src="https://user-images.githubusercontent.com/10009354/219107029-2b6a3df9-b690-4488-8125-babedf28b8cd.png">